docs/cog_guides/alias.rst

```diff
@@ -0,0 +1,270 @@
+assigned to this expression: ``ban {0} 7``
+
+.. image:: ../.resources/alias/example-2.png
+
+You can see in blue the "static" part of the alias, what is contained and
+doesn't need to be given, the orange part is the arguments given at the end of
+the command, just like a classic alias, and the green part is the positional
+argument we defined: the first argument of the alias will be the green part.
+
+You can add as much arguments as you want, they can start at ``{0}`` or ``{1}``
+and must be in order: ``{1}`` will be the first argument, ``{2}`` will be the
+second one...
+
+.. attention:: The numbers must be in order, you cannot use ``{0}`` and ``{2}``
+    without using ``{1}``.
+
+Here are more examples:
+
+*   *   Full command: ``[p]cleanup messages 75 True``
+    *   Alias: ``[p]alias add fullclear cleanup messages {0} True``
+    *   Invoked alias: ``[p]fullclear 75``
+
+    *The* ``True`` *at the end tells the bot to also clear pinned messages.*
+
+*   *   Full command: ``[p]repo add SinbadCogs
+        https://github.com/mikeshardmind/SinbadCogs v3``
+    
+    *   Alias: ``[p]alias add newrepo repo add {2} https://github.com/{1}/{2}``
+    *   Invoked with alias: ``[p]newrepo mikeshardmind SinbadCogs v3``
+
+**Arguments**
+
+* ``<alias_name>``: The new command name.
+
+* ``<command>``: The command to execute when ``[p]alias_name`` is invoked.
+
+.. _alias-command-alias-delete:
+
+""""""""""""
+alias delete
+""""""""""""
+
+.. note:: |mod-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]alias [delete|remove|del] <alias_name>
+
+**Description**
+
+Removes an alias from the list. Check the list with
+the :ref:`alias list <alias-command-alias-list>` command.
+
+**Arguments**
+
+* ``<alias_name>``: The alias' name to delete.
+
+.. _alias-command-alias-list:
+
+""""""""""
+alias list
+""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]alias list
+
+**Description**
+
+Shows all of the existing aliases on the current server.
+
+.. _alias-command-alias-show:
+
+""""""""""
+alias show
+""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]alias show <alias_name>
+
+**Description**
+
+Shows the command associated to the alias.
+
+**Arguments**
+
+* ``<alias_name>``: The alias you want information from.
+
+.. _alias-command-alias-help:
+
+""""""""""
+alias help
+""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]alias help <alias_name>
+
+**Description**
+
+Shows help message for an alias.
+
+**Arguments**
+
+* ``<alias_name>``: Alias you want to get help from.
+
+.. _alias-command-alias-global:
+
+""""""""""""
+alias global
+""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]alias global
+
+**Description**
+
+Another group command which contains the :ref:`add
+<alias-command-alias-add>`, :ref:`del
+<alias-command-alias-delete>` and :ref:`list
+<alias-command-alias-list>` commands.
+
+They work the same, except the created aliases will be
+global instead of being server-wide.
```

Comment:
Not necessarily needs changing, but I would say "just server-wide" for clarity as when I read over this i had to re-read.
```suggestion
global instead of being just server-wide.
```